### ISSUE TITLE
index-servers option in setup.cfg not working

### ISSUE THREAD

**Issue by wesleybl** [zestsoftware/zest.releaser#436]
I have a project with the following `setup.cfg`:

```
[zest.releaser]
extra-message = [skip ci]

[distutils]
index-servers =
    nexus
```

But when I run `fullrelease` on this project, I am asked if I want to upload it to all the servers that I have registered in `.pypirc`.

I'm using `zest.releaser 9.0.0` and `python 3.11`. It seems that in version `8.0.0` this worked.

**Comment by wesleybl** [comment]
@mauritsvanrees does this commit have anything to do with this?

https://github.com/zestsoftware/zest.releaser/commit/8aed13ae84c0206569ad4026c6ada6a83994bc2d

**Comment by wesleybl** [comment]
> BTW, I have six index-servers in my ~/.pypirc, one of them named pypi, and in my bash config I have export TWINE_REPOSITORY=pypi. Then zest.releaser only uses this single index server, also with my new code.

I tried in 9.0.0:

```
TWINE_REPOSITORY=nexus fullrelease
```

But it still asks for other repositories.

**Comment by wesleybl** [comment]
I tried it again and can confirm that 

```
TWINE_REPOSITORY=nexus fullrelease
```
works. Cool!

**Comment by mauritsvanrees** [comment]
Fix released in 9.1.1.  Thanks for the report!
